### DISCUSSION TITLE
C++ Implementation of Jaccard Similiartiy

### DISCUSSION THREAD

**Discussion by JerryXin2** [OlivierBinette/StringCompare#22] [Q&A]
Me and Garrett were wondering about how we could import the token class and declare it in C++: after we know how to do that, we both feel like we could write the C++ code.

**Comment by OlivierBinette** [comment]
@JerryXin2 I haven't implemented the Tokenizer class in C++ yet, so we'd have to do that first. You can give it a shot or otherwise I'll work on it this weekend.
